msg/en_US/utils.msg

```diff
@@ -978,9 +978,9 @@ usage: %1$s applyinfo [OPTION] database-name\n\
 \n\
 valid options:\n\
   -r, --remote-host-name      remote host name; display remote node's active log information \n\
-  -a, --applied-info 	      display applied information \n\
+  -a, --applied-info 	      display applied information; the -L option is required \n\
   -L, --copied-log-path=PATH  path of copied log volumes; display copied log information \n\
-  -p, --page=ID               page id; default : 0\n\
+  -p, --page=ID               page id; default : 0; the -L option is required \n\
```

Comment:
For utility msgs, the default value is usually shown at the end of the description. Isn't it?

Reply:
> To merge to the develop, other languages than en_US also have to be supported.

Right. I will change other languages msg file.

Reply:
Thank you. your comment seems so correct.